src/components/main-site/Header.js

```diff
@@ -12,36 +12,43 @@ import { faUserCircle } from "@fortawesome/free-solid-svg-icons";
 import * as ROUTES from "../../constants/routes";
 
 import img from "../../assets/img/header.jpg";
+import logo from "../../assets/img/logo_resized.png"
+import UPEclass from "../../assets/img/UPEclass.jpg"
+
 
 const styles = {
   mainNav: {
     paddingTop: "10px",
+    paddingLeft: "200px",
     paddingBottom: "10px",
-    fontfamily: "Andale Mono, monospace",
-    borderTop: "3px solid #f21131",
+    fontfamily: "Recursive",
+    borderTop: "3px solid #871a13", // was f21131
```

Comment:
Might be a good time to make a styles class that defines colors like in inquisitor (we can do that in a separate PR though)